src/app/business/block/fileShare.component.html

```diff
@@ -66,29 +66,20 @@
             </form-item>
             <form-item [required]="true" label="{{I18N.keyID['sds_fileShare_acl_user']}}">
                 <div class="ui-grid-row volume-item" *ngFor="let AclsItem of aclsItems;index as i">
-                    <!-- <p-dropdown [options]="availabilityip" formControlName="{{'user'+i}}" [style]="{'min-width':96}"></p-dropdown> -->
                     <div class="ui-grid-col-9" style="margin-top: -10px">
-                        <input type="text" pInputText formControlName="{{'userInput'+i}}"/>
-                        <div class="ui-message-container" *ngIf="createAclsFormGroup.controls['userInput'+i] && !createAclsFormGroup.controls['userInput'+i].valid&&createAclsFormGroup.controls['userInput'+i].dirty">
+                        <input type="text" pInputText formControlName="{{'userInput'+i}}" (keyup)="userIpChnage(i)"/>
```

Comment:
fix typo

Reply:
I have just modified it. Please check it

Reply:
I still see that the function name is not changed.  
(keyup)="userIpChnage(i)" -> (keyup)="userIpChange(i)"